[CI] GenerativeMetricsIT test failing

**Build Scans:**
- [elasticsearch-intake #30447 / part3](https://gradle-enterprise.elastic.co/s/b6bvunydv6i6q)
- [elasticsearch-pull-request #100546 / part-3](https://gradle-enterprise.elastic.co/s/3dftq3njleuoy)
- [elasticsearch-pull-request #100314 / part-3](https://gradle-enterprise.elastic.co/s/66442ftccwy24)
- [elasticsearch-pull-request #100271 / part-3](https://gradle-enterprise.elastic.co/s/wr5gynusvlho4)
- [elasticsearch-pull-request #100141 / part-3](https://gradle-enterprise.elastic.co/s/v7al5frhvsvkk)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:qa:server:single-node:javaRestTest" --tests "org.elasticsearch.xpack.esql.qa.single_node.GenerativeMetricsIT.test" -Dtests.seed=A057B94B474E2573 -Dtests.locale=zh-Hans-MO -Dtests.timezone=UCT -Druntime.java=25
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.qa.single_node.GenerativeMetricsIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: query: ts k8s-downsampled,k8s-downsampl*,k8s* | eval  network.eth0.currently_connected_clients = network.cost + network.bytes_in, aoRQZsexLji = length(cluster), `event_city` = "IxVPkOsvce" | where  NOT true | stats  hkHujjeul = max(network.eth0.currently_connected_clients) + max(network.eth0.currently_connected_clients), `network.bytes_in` = max(network.cost) + avg(network.cost) + median(network.cost), LZTSAJTEre = count(network.bytes_in) + median(network.bytes_in) by event_log
exception: method [POST], host [http://[::1]:41101], URI [/_query?format=yaml&pretty=true&error_trace=true], status line [HTTP/1.1 400 Bad Request]
---
error:
  root_cause:
  - type: "verification_exception"
    reason: "Found 1 problem\nline 1:185: Output has changed from [[hkHujjeul{r}#29275,\
      \ network.bytes_in{r}#29280, LZTSAJTEre{r}#29284, event_log{f}#29291]] to [[hkHujjeul{r}#29275,\
      \ network.bytes_in{r}#29280, LZTSAJTEre{r}#29284, event_log{r}#29291]]. "
    stack_t
[truncated]
```

**Issue Reasons:**
- [main] 5 failures in test test (1.7% fail rate in 289 executions)
- [main] 4 failures in step part-3 (2.3% fail rate in 171 executions)
- [main] 4 failures in pipeline elasticsearch-pull-request (2.4% fail rate in 170 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -447,9 +447,6 @@ tests:
 - class: org.elasticsearch.xpack.esql.qa.mixed.FieldExtractorIT
   method: testByteFieldWithIntSubfieldTooBig {NONE}
   issue: https://github.com/elastic/elasticsearch/issues/137034
-- class: org.elasticsearch.xpack.esql.qa.single_node.GenerativeMetricsIT
-  method: test
-  issue: https://github.com/elastic/elasticsearch/issues/137071
 - class: org.elasticsearch.xpack.ml.integration.DatafeedJobsIT
   method: testDatafeedTimingStats_DatafeedRecreated
   issue: https://github.com/elastic/elasticsearch/issues/137207

--- a/x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/generative/GenerativeRestTest.java
+++ b/x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/generative/GenerativeRestTest.java
@@ -81,7 +81,8 @@ public abstract class GenerativeRestTest extends ESRestTestCase implements Query
         "time_series aggregate.* can only be used with the TS command",
         "implicit time-series aggregation function .* doesn't support type .*",
         "INLINE STATS .* can only be used after STATS when used with TS command",
-        "cannot group by a metric field .* in a time-series aggregation"
+        "cannot group by a metric field .* in a time-series aggregation",
+        "Output has changed from \\[.*\\] to \\[.*\\]" // https://github.com/elastic/elasticsearch/issues/134794
     );
 
     public static final Set<Pattern> ALLOWED_ERROR_PATTERNS = ALLOWED_ERRORS.stream()

